src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/AlwaysHasCom.kt

```diff
@@ -1,5 +1,5 @@
 package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity
 
 interface AlwaysHasCom {
-  var responsibleCom: CommunityOffenderManager
+  fun getCom(): CommunityOffenderManager
 }
```

Comment:
Lazy loading was not working for this approach, I think Hibernate proxies could not associate the object in the interface with the methods in licence abstraction : 
```
abstract fun getCom(): CommunityOffenderManager?
 abstract fun setCom(com: CommunityOffenderManager)
```